EATSSU/App/Sources/Presentation/Splash/NoticeSplashViewController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  NoticeSplashViewController.swift
+//  EATSSU-DEV
+//
+//  Created by 황상환 on 3/25/25.
+//
+
+import Foundation
+import UIKit
```

Comment:
`UIKit` 임포트 시, `Foundation + UI 관련 프레임워크` 됩니다.
```suggestion
import UIKit
```
Foundation 임포트는 생략하셔도 무방합니다!

Reply:
이것도 수정하겠습니다!